.github/workflows/erlang.yml

```diff
@@ -0,0 +1,39 @@
+name: Erlang CI
+
+on:
+  push:
+  pull_request:
+
+jobs:
+
+  build:
+
+    runs-on: ubuntu-latest
+    strategy: 
+      fail-fast: false
+      matrix:
+        erlang: [ 21,22,23 ]
```

Comment:
Is there already support for 24.0? We can probably drop 21.

Reply:
Yes, the workflow passes for Erlang 24, too. I added it.